actor/v7action/cloud_controller_client.go

```diff
@@ -129,6 +129,7 @@ type CloudControllerClient interface {
 	MakeRequestSendReceiveRaw(Method string, URL string, headers http.Header, requestBody []byte) ([]byte, *http.Response, error)
 	MapRoute(routeGUID string, appGUID string) (ccv3.Warnings, error)
 	PollJob(jobURL ccv3.JobURL) (ccv3.Warnings, error)
+	PollJobForStatus(jobURL ccv3.JobURL, state constant.JobState) (ccv3.Warnings, error)
```

Comment:
Given a job has a "state" and not a "status", it might be clearer to call this `PollForJobState()`, but it's quite common to use the words interchangeably so this is not a big one.

Reply:
That is a good point. Thanks for spotting it